src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -469,13 +470,13 @@ class EducationWizard extends React.Component {
               this.getButton('5495')}
             {newBenefit === 'yes' &&
               serviceBenefitBasedOn === 'other' &&
-              sponsorDeceasedDisabledMIA === 'yes' &&
-              this.getButton('5490')}
+              sponsorTransferredBenefits === 'yes' &&
+              this.getButton('1990E')}
```

Comment:
Note: /education/apply-for-education-benefits/application/1990E is the online tool that correlates to 22-1990E.